src/defaultV2-theme/blocks/cover.scss

```diff
@@ -92,12 +92,13 @@
 }
 
 .sv-cover__title .sd-title {
+  --default-font-size: var(--sjs-font-surveytitle-size, calc(2 * (var(--sjs-font-size, 16px))));
   margin: 0;
   color: $cover-title-forecolor;
   font-family: $font-surveytitle-family;
-  font-size: $font-surveytitle-size;
+  font-size: var(--default-font-size);
```

Comment:
It was $font-surveytitle-size: var(--sjs-font-surveytitle-size, calc(2 * var(--sjs-font-size, $font-size)))
It seems like --sjs-font-surveytitle-size has been lost